caffe2/opt/converter.cc

```diff
@@ -11,7 +11,7 @@ using namespace nom;
 namespace {
 
 std::vector<int> getStrides(std::map<std::string, caffe2::Argument> argMap) {
-  std::vector<int> strides;
+  std::vector<int> strides = {1, 1};
```

Comment:
Why changing the default value?

Reply:
to get the expected filter format, the expected_weights_descriptor API in optimize_ideep requires the init values for all parameters.